src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,31 @@
+.stars {
+  display: flex;
+  column-gap: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.stars--0 .stars__star {
+  background-image: url(../images/star.svg);
+}
+
+.stars--5 .stars__star,
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(n + 2),
+.stars--2 .stars__star:nth-child(n + 3),
+.stars--3 .stars__star:nth-child(n + 4),
+.stars--4 .stars__star:nth-child(n + 5) {
+  background-image: url(../images/star.svg);
+}
```

Comment:
Similar to the previous comment, this block of code is setting the background image to `star.svg` for the non-active stars in various `stars--#` blocks. This is redundant because the `stars__star` class already sets the background image to `star.svg`. It's better to only set the `background-image` property for the active stars in the `stars--#` blocks and remove this block of code entirely.